README.md

```diff
@@ -958,6 +958,106 @@ services:
 
 </details>
 
+### How to use a docker registry to cache images layer
+
+<details>
+
+<summary>Read the cookbook</summary>
+
+You can use a docker registry to cache images layer, it can be useful to speed up the build process during the CI and 
+local development.
+
+First you need a docker registry, in following examples we will use the GitHub registry (ghcr.io).
+
+Then add the registry to the context variable of the `castor.php` file:
+
+```php
+function create_default_variables(): Context
+{
+    return [
+        // [...]
+        'registry' => 'ghcr.io/your-organization/your-project',
+    ];
+}
+```
+
+Once you have the registry, you can push the images to the registry:
+
+```bash
+castor docker:push
+```
+
+This command will generate a bake file with the images to push from the `cache_from` directive of the `docker-compose.yml` file.
+If you want to add more images to push, you can add the `cache_from` directive to them.
+
+```yaml
+services:
+    my-service:
+        build:
+            cache_from:
+                - "type=registry,ref=${REGISTRY:-}/my-service:cache"
+```
+</details>
+
+### How to use cached images in a GitHub action
```

Comment:
```suggestion
#### How to use cached images in a GitHub action
```

Reply:
This one is correct (it's another entry in the cookbook)